### DISCUSSION TITLE
Выбор дизайна веток комментариев

### DISCUSSION THREAD

**Discussion by jarvis394** [jarvis394/geekr#171] [Ideas]
### Какой нравится больше и почему?

<img src="https://user-images.githubusercontent.com/37776763/156637742-358d74fb-b860-49ba-84d4-96c8ad3bade7.png" width="700" />

Голосуйте за **первый вариант** (слева) с помощью реакции "👍", а за **второй** (справа) – "👎". Также было бы неплохо в комментариях написать о своём выборе ✨

**Comment by centralhardware** [comment]
Трудно найти различия, правый так как можно визуально понять уровень вложенности, но по честному не думаю что кто то бы заметил разницу если не ставить 2 варианта рядом

**Reply by jarvis394** [reply]
В первом варианте сложно нажимать на линии с мобильных устройств (хотя спорно), зато можно скрыть любую ветку из любой части вложенности комментариев. Во второй версии на кнопочки нажимать легче, но можно скрыть только одну ветку из одного комментария. То есть нельзя, доскроллив до 4 комментария в примере, скрыть ветку первого комментария, который уже ушёл из viewport'a

**Reply by centralhardware** [reply]
Тогда я лично за первый вариант
